packages/suite/src/components/firmware/Img.tsx

```diff
@@ -35,7 +35,7 @@ export const WarningImg = () => (
 );
 
 export const ErrorImg = () => (
```

Comment:
It seems to me that this picture is not used anywhere at all.

Reply:
It does not look like used anywhere. I can see it is imported in `packages/suite/src/components/firmware/index.tsx` and there are other images imported there that do not look like they are used anywhere else. But I might be missing something.

```javascript
export {
    InitImg,
    SuccessImg,
    ErrorImg,
    DisconnectImg,
    WarningImg,
    ConnectInNormalImg,
    ConnectInBootloaderImg,
    SeedImg,
} from './Img';
```

Reply:
@matejkriz I removed file `packages/suite/src/components/firmware/Img.tsx` and the re-export of them since they are not used anywhere.